poetry.lock

```diff
@@ -1,24 +1,34 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
+
 [[package]]
 name = "amqp"
-version = "5.0.9"
```

Comment:
Wooo, I haven't noticed at first that this file changed a lot.
Why did you bump all lib?
Lib bump should not be part of this PR.
Could you please reset this file and add your required psql lib?
Thx

Reply:
Just saw that. I'm not an poetry expert, maybe you should fix versions on pyproject.toml ? (only minimal version are set using "^").
Anyway, I'll revert poetry.lock today